optimize/apis-tools/optimize-api/tutorial.md

```diff
@@ -0,0 +1,167 @@
+---
+
+In this tutorial, we'll step through examples to highlight the capabilities of the Optimize API, such as listing your existing dashboard IDs, or deleting a dashboard.
+
+## Getting started
+
+- You need authentication to access the API endpoints. Find more information [here](./optimize-api-authentication.md).
+- To properly execute the commands to list existing dashboard IDs and delete a dashboard, ensure you have [created a collection](../../components/userguide/collections-dashboards-reports.md) containing a [dashboard](../../components/userguide/creating-dashboards.md).
+
+## Prerequisites
+
+- If you haven't done so already, [create a cluster]($docs$/guides/assets/react-components/create-cluster).
+- Upon cluster creation, [create your first client]($docs$/guides/setup-client-connection-credentials). Ensure you check the Optimize client scope box.
+
+:::note
+Make sure you keep the generated client credentials in a safe place. The **Client secret** will not be shown again. For your convenience, you can also download the client information to your computer.
+:::
+
+- In this tutorial, we utilize a JavaScript-written [GitHub repository](https://github.com/camunda/camunda-api-tutorials) to write and run requests. Clone this repo before getting started.
+- Ensure you have [Node.js](https://nodejs.org/en/download) installed as this will be used for methods that can be called by the CLI (outlined later in this guide). Run `npm install` to ensure you have updated dependencies. Run `npm install` to ensure you have updated dependencies.
+
+## Set up authentication
+
+To get started, examine the `auth.js` file in the GitHub repository. This file contains a function named `getAccessToken` which executes an OAuth 2.0 protocol to retrieve authentication credentials based on your client id and client secret. We will call this function whenever we need an authentication token for an API request.
+
+:::note
+You may notice in the `auth.js` file that `getAccessToken` takes two arguments. This separates the authentication between the Administration API and the other component APIs (in this case, the component being Optimize).
+:::
+
+1. To set up your credentials, create an `.env` file which will be protected by the `.gitignore` file. These keys will be consumed by the `auth.js` file to execute the OAuth protocol, and should be saved when you generate your client credentials in [prerequisites](#prerequisites).
+2. Examine the existing `.env.example` file for an example of how your `.env` file should look upon completion. You will need to add your `COMPONENTS_CLIENT_ID`, `COMPONENTS_CLIENT_SECRET`, `OPTIMIZE_BASE_URL`, and `OPTIMIZE_AUDIENCE`, which is `optimize.camunda.io` in a Camunda 8 SaaS environment. Do not place your credentials in the `.env.example` file, as this example file is not protected by the `.gitignore`.
```

Comment:
> and `OPTIMIZE_AUDIENCE`, which is `optimize.camunda.io`

@toco-cam I had an `https://` before this in the example env file in the tutorial, which was causing this to fail. I fixed this in the repo, and it works for me now. Let me know if you're still having issues, and thanks for catching!

Reply:
Oops. I published this before I saw the message from @pepopowitz above 😅 

Regardless, let us know if you're still having trouble!

Reply:
We won't have any users working in an ultrawombat environment, so I do wonder if your issues are coming from your .env file -- specifically, Steve mentioned your client ID/secrets may no longer be active.

If you try our tutorial from a personal setup at camunda.io with updated cluster/client details, do you run into the same issues?

Reply:
@christinaausley: Ok, it was most likely the ultrawombat environment. I got it working on https://bru-2.optimize.camunda.io.

But the error message was strange - it seemed as Ultrawombat is using different text encoding.

Reply:
I suspect the text encoding message is more a symptom of the issue, than the issue itself. Those types of messages often occur when an HTML error page is being served and the caller tries to parse it as JSON.

Here's what I think was happening:

* You created the API client in the ultrawombat environment
* The "authentication" endpoint [is hardcoded in the tutorial](https://github.com/camunda/camunda-api-tutorials/blob/ec7e97a91d911344c9898e275c07917a71a68e89/auth.js#L41) to the production SaaS auth URL
* You were getting an error page when attempting to authenticate, because the API keys weren't known to the production environment

This does raise the question -- should the auth URL be hardcoded in the sample code, or should it be configurable by an environment variable?